### ISSUE TITLE
The original dataset's transform is performed only once before data are sent to workers.

### ISSUE THREAD

**Issue by Ethanwl** [OpenMined/PySyft#4650]
## Question
My codes:
`dataset = dataset.federate([worker[0]])`
`loader = sy.FederatedDataLoader(dataset, shuffle=True, batch_size=batch_size)`
According to the codes:
https://github.com/OpenMined/PySyft/blob/f776c0b8ead2d3bc04a9ddcb80a0f0c36fe030ca/syft/frameworks/torch/fl/dataset.py#L282-L291
When I call 'dataset.federate( )', the original dataset's transform is performed only once before data are sent to workers. When the data are sampling at workers, transform is not performed again. Is my understanding right?
But when RandomCrop and RandomHorizontalFlip are included in transform, performing transform only once may be a problem.
How can transform be performed at workers before every sampling like in Pytorch?

**Comment by zwvews** [comment]
My method is to overwrite getitem of Basedataset.

**Comment by cereallarceny** [comment]
Hello! Just letting you know that we are no longer planning on supporting anything on the 0.2.x product line and that all work should be ported over to 0.3.x, which is considered a complete rebuild of PySyft. Because of that, I’ll be closing this issue. If you feel this is a mistake, or if the issue actually applies to 0.3.x as well, please feel free to ping me on Slack and I’ll reopen the issue.
